extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -101,4 +111,24 @@ private function formatTitle(Content $content) {
 
 		return $content->getTitle();
 	}
+
+	private function onValidWiki() {
+		global $wgCityId;
+
+		return in_array(
+				$wgCityId,
+				[
+						147,    // starwars
+						130814, // gameofthrones
+						3035,   // fallout
+						2237,   // dc
+						2233,   // marvel
+						208733, // darksouls
```

Comment:
Yah, I added it because I see we have a ticket to do it. I figured since we're going to add it it's better to get ahead of that now rather than having to push out another MW release.